src/Layers/FeatureLayer/FeatureGrid.js

```diff
@@ -21,6 +21,7 @@ export var FeatureGrid = L.Layer.extend({
 
   onRemove: function () {
     this._map.removeEventListener(this.getEvents(), this);
+    delete this._map;
     this._removeCells();
```

Comment:
I'm curious what, this._removeCells() gives you if you are removing the map listeners and then the map?